tests/UnifiedSpecTests/EntityMap.php

```diff
@@ -29,19 +29,17 @@
 
 class EntityMap implements ArrayAccess
 {
-    /** @var array */
-    private $map = [];
+    private array $map = [];
 
     /**
      * Track lsids so they can be accessed after Session::getLogicalSessionId()
      * has been called.
      *
-     * @var stdClass[]
+     * @var list<stdClass>
```

Comment:
This is incorrect. The `set()` method assigns the lsid object with a string key.

```suggestion
     * @var array<string, stdClass>
```

Reply:
Oh, I missed that. Thanks for catching it!